src/UriNormalize.c

```diff
@@ -245,6 +247,23 @@ static URI_INLINE void URI_FUNC(LowercaseInplace)(const URI_CHAR * first,
 
 
 
+static URI_INLINE void URI_FUNC(LowercaseInplaceAfterPercentEncoding)(const URI_CHAR * first,
```

Comment:
Could you name the function `LowercaseInplaceExceptPercentEncoding` with "Except" rather than "After"? It's close but takes takes time out of the picture.